core/services/ardupilot_manager/typedefs.py

```diff
@@ -57,10 +57,10 @@ class SITLFrame(str, Enum):
     UNDEFINED = " undefined"
 
 
-def get_sitl_platform_name() -> str:
+def get_sitl_platform_name(machine_arch: str) -> str:
     """Get SITL platform name based on machine architecture."""
 
-    if machine() == "x86_64":
+    if machine_arch == "x86_64":
```

Comment:
since this receives a string as input, we would be more safe if we check if `machine_arch` contains `arm`, since it's a more generic check for arm and arm64, it'll also make more simpler and less error prone to detect x86_64 arch if the string is different.

Reply:
This is just a minor request, everything else looks ready to approve.

Reply:
Good idea